src/Extension/AbstractOrderedExtension.php

```diff
@@ -31,12 +32,24 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     }
 
     /**
+     * @param OptionsResolver $resolver
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults(array('position' => null));
+
+        if (method_exists('Symfony\Component\Form\AbstractType', 'configureOptions')) {
```

Comment:
But then, what can I check in the OptionsResolver in order to avoid deprecated calls?